incuna-sass/mixins/_sprites.sass

```diff
@@ -1,5 +1,5 @@
 $sprites-1x: sprite-map("1x/sprite-files/*.png", $layout: smart)
-$sprites-2x: sprite-map("2x/sprite-files/*.png", $spacing: 5px)
+$sprites-2x: sprite-map("2x/sprite-files/*.png", $layout: smart)
```

Comment:
@perry Is there a reason the settings differed between 1x and 2x? It's been like that since the beginning (2ae65823240f71120c5d667d8a58c624b2948236)